app/src/main/java/timur/gilfanov/messenger/domain/usecase/user/repository/ChangeLanguageRepositoryError.kt

```diff
@@ -1,29 +1,58 @@
 package timur.gilfanov.messenger.domain.usecase.user.repository
 
+import timur.gilfanov.messenger.domain.entity.user.Settings
+
 /**
- * Errors specific to language change repository operations.
- *
- * Defines operation-specific errors returned by the repository layer
- * when changing language preferences.
- *
- * ## Operation-Specific Errors
- * - [LanguageNotChangedForAllDevices] - Partial synchronization failure
+ * Errors that can occur during language change repository operations.
  *
- * ## Common Errors
- * - [UserRepository] - Wraps user-related repository errors
+ * Represents failures when changing user language preferences at the repository layer.
  */
 sealed interface ChangeLanguageRepositoryError {
     /**
-     * Language preference was updated on some devices but not all.
+     * The language preference was not changed.
+     *
+     * @property transient Indicates if the error is transient (retriable) or permanent
+     */
+    data class LanguageNotChanged(val transient: Boolean) : ChangeLanguageRepositoryError
```

Comment:
May be use Failed to be consistent with sync settings error?
```suggestion
    data class Failed(val transient: Boolean) : ChangeLanguageRepositoryError
```

Reply:
No, LanguageNotChanged fit better here